doctests/dt-json.js

```diff
@@ -0,0 +1,382 @@
+// EXAMPLE: json_tutorial
+// HIDE_START
+import assert from 'assert';
+import {
+  createClient
+} from 'redis';
+
+const client = await createClient();
+await client.connect();
+// HIDE_END
+// REMOVE_START
+await client.flushDb();
+// REMOVE_END
+
+// STEP_START set_get
+const res1 = await client.json.set("bike", "$", '"Hyperion"');
+console.log(res1); // OK
+
+const res2 = await client.json.get("bike", "$");
+console.log(res2); // "Hyperion"
+
+const res3 = await client.json.type("bike", "$");
+console.log(res3); //  [ 'string' ]
+// STEP_END
+
+// REMOVE_START
+assert.equal(res2, '"Hyperion"');
+// REMOVE_END
+
+// STEP_START str
+const res4 = await client.json.strLen("bike", "$");
+console.log(res4) //  [10]
+
+const res5 = await client.json.strAppend("bike", '" (Enduro bikes)"');
+console.log(res5) //  27
+
+const res6 = await client.json.get("bike", "$");
+console.log(res6) //  ['"Hyperion"" (Enduro bikes)"']
+// STEP_END
+
+// REMOVE_START
+assert.equal(res6, ['"Hyperion"" (Enduro bikes)"']);
+// REMOVE_END
+
+// STEP_START num
+const res7 = await client.json.set("crashes", "$", 0);
+console.log(res7) //  True
```

Comment:
we get back OK which seems right, not True.  doesn't seem that different from line 16